src/main/java/org/cactoos/func/FallbackFrom.java

```diff
@@ -37,7 +38,7 @@
  * @param <T> Type of result
  * @since 0.31
  */
-public final class FallbackFrom<T> implements Func<Throwable, T> {
+public final class FallbackFrom<T> implements Fallback<T> {
```

Comment:
@baudoliver7 lets move this class as an inner class of Fallback and rename it just `From`.